src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -0,0 +1,246 @@
+<template>
+  <div id="amalgamating-businesses">
+    <v-btn
+      id="btn-add-amalgamating-business"
+      outlined
+      color="primary"
+      class="btn-outlined-primary"
```

Comment:
This is added to have the special color when button is disabled.

Reply:
When is this ever disabled?

Reply:
NVM - when you're prompting for a business name, the add buttons are disbabled.

Reply:
Did we add this class elsewhere? What does the button look like without it?

Reply:
Yes Sev! https://github.com/bcgov/business-create-ui/blob/5e99b3750fbf02855270417a9dd544431bf0c350/src/components/common/PeopleAndRoles.vue#L163

Left side is with it while left side is without it:
![with and without](https://github.com/bcgov/business-create-ui/assets/122301442/52ff57d7-05b8-4d4f-b0d1-77dc8af73e0f)

Reply:
Great! I forgot about that. Good job :)